### DISCUSSION TITLE
How do I keep LossLess from closing after I export a clip?

### DISCUSSION THREAD

**Discussion by sujfuy** [mifi/lossless-cut#2089] [General]
I like this program better than anything I've found for the purpose of trimming and cutting out segments of videos, but I don't like having to reopen the program, reopen the previous file and locate where I left off.  Surely there's a way to stop the program from closing after each exporting of a clip.

**Comment by mifi** [comment]
https://github.com/mifi/lossless-cut/discussions/2034#discussioncomment-10237094
